backend/Rhyolite/Backend/Listen.hs

```diff
@@ -276,4 +276,103 @@ deleteAndNotify :: forall n a m.
   => Id a -> m ()
 deleteAndNotify aid = do
   deleteBy (fromId aid :: Key a BackendSpecific)
-  notify NotificationType_Delete (notification aid) aid
\ No newline at end of file
+  notify NotificationType_Delete (notification aid) aid
+
+
+data Change a = Change
```

Comment:
It doesn’t seem possible to have both old and new be `Nothing`. Wouldn’t `These` be a better fit?